Default container name have been changed

PeterSelchDahl opened the pull request:
"insights-logs-networksecuritygroupflowevent" container is renamed to "insights-logs-networksecuritygroupevent". This should also be changed in the PowerBI NSG template.

azurecla commented:
Hi __@stefsch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stefsch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PeterSelchDahl committed:
Default container name have been changed

"insights-logs-networksecuritygroupflowevent" container is renamed to "insights-logs-networksecuritygroupevent". This should also be changed in the PowerBI NSG template.
